### DISCUSSION TITLE
Marketing emails

### DISCUSSION THREAD

**Discussion by nandorojo** [sofn-xyz/mailing#176] [Q&A]
It looks like Mailing is primarily targeted towards [transactional emails](https://postmarkapp.com/transactional-email) sent via SMTP. That said, has anyone here experimented with using Mailing as a **marketing email** template maker?

I want to use Mailing for our changelog updates to match our website's aesthetic. This will also allow pulling in our CMS from an API route, so that our website and emails have the same exact data.

It doesn't look like email clients like Mailchimp allow you to send custom HTML emails via API unless it's for transactional emails. Does anyone know of a marketing email client that will allow this?

Should I just send transactional emails and include an unsubscribe link? Ideally I can use something like a Mailchimp campaign + my custom email code generated by Mailchimp to get the best of both worlds.

If I find a solution, I'll share it here.

Thanks!

**Comment by mrlubos** [comment]
Hey @nandorojo, are you able to describe which limitations do you foresee with using mailing for marketing campaigns? You should be able to use any CMS you want to generate content for your emails

**Reply by nandorojo** [reply]
Yeah, I tried using Mailchimp’s transactional product, which uses Mandrill under the hood. However, these emails have the following limitations:

1. Opens/clicks don’t show up on the Mailchimp dashboard. Instead they go to the Mandrill transactional emails. With non-API Mailchimp, you can create campaigns here and track the responses within a campaign.
2. Unsubscribe flows aren’t handled through the Mailchimp website. I’ll probably have to build my own unsubscribe page and store opt-outs in the database somewhere. Not the end of the world I suppose.

The benefit of using something like Mailchimp is that it lets you visualize open rates & unsubscribes etc in a nice UI. However, I haven’t seen a marketing email provider like Mailchimp with a good API for sending custom emails, so I might just use Postmark directly and forward data to Segment.

(I’m on my phone so sorry if this rambled on a little)
